### DISCUSSION TITLE
Strange padding-bottom on q-img element

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#13745] [General - Components / Directives / etc]
I have a `q-img` inside a `q-card-section` element.

It looks like this in my app code:

```Vue
<q-card-section>
  <div class="flex row items-center">
    <div class="text-h5">Billing</div>
    <q-img class="q-ml-sm" src="../assets/powered-by-stripe-blurple.svg" style="max-width: 100px" />
  </div>
</q-card-section>
```

But it outputs the image with too much padding. You can see a screenshot here: https://imgur.com/a/RtS3ziX

This is what gets computed in the browser. Strangely it adds this `<div style="padding-bottom: 200%;"></div>` which I think is the problem:

```HTML
<div class="q-img q-img--menu q-ml-sm" style="max-width: 100px;" role="img">
  <div style="padding-bottom: 200%;"></div>
  <div class="q-img__container absolute-full">
    <img class="q-img__image q-img__image--with-transition q-img__image--loaded" style="object-fit: cover; object-position: 50% 50%;" loading="lazy" fetchpriority="auto" aria-hidden="true" draggable="false" src="/src/assets/powered-by-stripe-blurple.svg">
  </div>
  <div class="q-img__content absolute-full q-anchor--skip"></div>
</div>
```

What's going on here?

What have I done wrong?

**Comment by pratik227** [comment]
@BenJackGill 

Try this code . Make sure there is no padding in SVG.

```
<q-card class="my-card" flat bordered>

       <q-card-section horizontal>
        <q-card-section class="flex items-center">
          Billing
        </q-card-section>

        <q-img
          src="https://cdn.quasar.dev/img/parallax2.jpg"
        />
      </q-card-section>
    </q-card>
```

Codepen - https://codepen.io/Pratik__007/pen/OJQYOBa

Doc - https://quasar.dev/vue-components/card#horizontal

**Reply by BenJackGill** [reply]
Thanks for the help, and sorry for my late reply.

I have re-created my component in a Code Pen, modified using your advice: https://codepen.io/thecashbag/pen/YzaKLrx

It works fine on the Code Pen but when I use it on my app there is still the big gap caused by the padding. Here is a screenshot of what it now looks like in my app: https://imgur.com/a/QepN4BO

The Code Pen version has a div of `<div style="padding-bottom: 22.6667%;"></div>` and it looks fine.

But my app has a div of `<div style="padding-bottom: 200%;"></div>`. If I remove the div or change it to 22.6667% it looks fine. Not sure why my app gives it 200% though.

**Comment by metalsadman** [comment]
you should make a codepen, works fine like what pratik did.
